src/sass/style.scss

```diff
@@ -141,12 +141,58 @@ blockquote {
   font-size: Max(5vw, 2.5rem);
 }
 
+.page-intro {
+  font-size: Max(2vw, 1rem);
+}
+
+.branch-list {
```

Comment:
Do we need to have the new class names for the branch blocks if the post blocks and the branch blocks have the same stylings?